spec/rails_helper.rb

```diff
@@ -74,6 +74,7 @@
 
   config.include Devise::Test::IntegrationHelpers, type: :feature
   config.include Devise::Test::IntegrationHelpers, type: :request
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
what is the difference between `:request` and `:controller`? do the other controller specs need changing?

Reply:
https://github.com/jshawl/saas-starter/issues/62